app.territoiresentransitions.react/src/ui/dropdownLists/ActionsReferentielsDropdown/ActionsReferentielsDropdown.tsx

```diff
@@ -58,23 +62,25 @@ const ActionsReferentielsDropdown = ({
           referentiel = matches[1];
           identifiant = matches[2];
         }
-
         if (!identifiant) {
           const fuse = new Fuse(actionListeFiltered, {
             keys: ['nom'],
-            threshold: 0.5,
+            threshold: 0.3,
             shouldSort: false,
           });
 
-          actionListeFiltered = fuse.search(search).map((r) => r.item);
+          actionListeFiltered = fuse
+            .search(search)
+            .map((r: FuseResult<ActionItem>) => r.item);
         } else {
-          actionListeFiltered = actionListeFiltered.filter((action) => {
-            return (
-              action.identifiant.startsWith(identifiant) &&
-              (!referentiel ||
-                referentiel.toLowerCase() === action.referentielId)
-            );
-          });
+          actionListeFiltered = actionListeFiltered.filter(
+            (action: ActionItem) => {
```

Comment:
l'inférence ne fait pas le job ici ? 
```suggestion
            (action) => {
```

Reply:
Oui il me semble que l'inférence devrait suffire effectivement